tensor_genn/algorithms/relu_ann.py

```diff
@@ -69,37 +117,46 @@ def convert(self, tf_model):
         
         cs_init = {"magnitude":10.0}
 
+        # Fetch augmented weight matrices
+        g_weights, n_units = self.create_weight_matrices(tf_model)
+        gw_inds = [np.nonzero(gw) for gw in g_weights]
+        gw_vals = [g_weights[i][gw_inds[i]].reshape(-1) for i in range(len(g_weights))]
```

Comment:
You could replace the ``reshape(-1)`` with ``flatten()`` to make your intentions somewhat clearer. Also you could maybe move this logic into the ``create_weight_matrices`` method which would perhaps be a nice way of tidying up creating sparse and dense connectivity (``gw_inds[i]`` could perhaps be ``None`` if the matrix should be dense)